client/lib/abtest/active-tests.js

```diff
@@ -1,5 +1,13 @@
 /** @format */
 export default {
+	redesignedSidebarBanner: {
```

Comment:
Is this A/B test documented anywhere? As @shaunandrews pointed out, some upsells were not updated, so I wanted to make sure that will be taken into account when analyzing the results of this test.

Best to add this test to the end of the file - that's how the rest is ordered.